vsap-bios/copy-bios-to-usb.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
```

Comment:
Maybe add a `set -euo pipefail` to the start of each script to make sure we exit and notify on errors

Reply:
That approach will require quite a bit of refactoring since failed commands are used to determine whether or not to take certain actions, failures are ok, etc... It can be done, but I don't think it's worth it in this case.

Reply:
Ah gotcha, thanks for clarifying